lib/dolly/timestamps.rb

```diff
@@ -0,0 +1,22 @@
+require 'active_support/concern'
```

Comment:
ok, so oyu can use the `classMethods` block and use included in the Document class